.jenkins/sarus_run_test.sh

```diff
@@ -18,39 +18,36 @@ pip install -r ../requirements.txt
 python -c 'import fv3config; import yaml; fid=open("../tests/pytest/config/default.yml", "r"); config = yaml.safe_load(fid); fv3config.write_run_directory(config, "./c12_test")'
 deactivate
 
-cd ${PWD}/c12_test
+# Set the working directory for the upcoming SLURM jobs
+cd c12_test
 cp ../../.jenkins/job_jenkins_sarus .
-cp ../../.jenkins/md5*.txt .
+cp ../../tests/pytest/reference/circleci/default/md5.txt .
 export SCRATCH_DIR=${PWD}
 
 
-# Run a c12 regression test for each Docker image
-dockerfiles=( $( ls ../../docker/Dockerfile.* ) )
-for df in ${dockerfiles[@]}; do
-   # Grab the architecture tag from the dockerfile filename
-   arch=${df:24}
+# Run c12 regression test on each Docker image
+declare -a tags=("hpc" "hpc-serialize")
+for tag in "${tags[@]}"; do
```

Comment:
See above...
- I think it makes sense to define `tags` at the beginning of this shell script.
- Standard syntax for simple cases is `for tag in ${tags} ; do` and works just fine.

Reply:
reformatted do loop. Done